SWIG/ratehelpers.i

```diff
@@ -111,6 +116,9 @@ class DepositRateHelperPtr : public boost::shared_ptr<RateHelper> {
             return new DepositRateHelperPtr(
                 new DepositRateHelper(rate, libor));
         }
+		Real impliedQuote() {
```

Comment:
This (and the ones in the other classes) can go in the interface of the base `RateHelper` class instead.

Reply:
Yes indeed.  In any case I have not (yet) managed to properly export the `setTermStructure(YieldTermStructure*)` function, without which `impliedQuote` will not be useful anyways.

Reply:
I'm not sure that `setTermStructure` should be exported.  It is used by the bootstrap algorithm, and not really meant to be used to interact with the helpers directly...